### PR TITLE
Add basic examples for `install-powershell.ps1`

### LINKED ISSUE #10911: install-powershell-readme.md doesn't contain any instructions to install from PowerShell
The install-powershll-readme.md doesnt show any powershell specific instructions even though there is a PowerShell script available in this folder.

### PR DESCRIPTION
**Pull request by kilasuit**
Adds some basic details for install-powershell.ps1 script

Fixes #10911 

# PR Summary

Adds some basic details to install-powershell-readme.md for the install-powershell.ps1 script

## PR Checklist

- [x] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
    - Use the present tense and imperative mood when describing your changes
- [x] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] [Make sure all `.h`, `.cpp`, `.cs`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
    - If the PR is work in progress, please add the prefix `WIP:` or `[ WIP ]` to the beginning of the title (the `WIP` bot will keep its status check at `Pending` while the prefix is present) and remove the prefix when the PR is ready.
- **[Breaking changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)**
    - [x] None
    - **OR**
    - [ ] [Experimental feature(s) needed](https://github.com/MicrosoftDocs/PowerShell-Docs/blob/staging/reference/6/Microsoft.PowerShell.Core/About/about_Experimental_Features.md)
        - [ ] Experimental feature name(s): 
- **User-facing changes**
    - [ ] Not Applicable
    - **OR**
    - [x] [Documentation needed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
        - [x] Issue filed: 
- **Testing - New and feature**
    - [x] N/A or can only be tested interactively
    - **OR**
    - [ ] [Make sure you've added a new test if existing tests do not effectively test the code changed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#before-submitting)
- **Tooling**
    - [x] I have considered the user experience from a tooling perspective and don't believe tooling will be impacted.
    - **OR**
    - [ ] I have considered the user experience from a tooling perspective and enumerated concerns in the summary. This may include:
        - Impact on [PowerShell Editor Services](https://github.com/PowerShell/PowerShellEditorServices) which is used in the [PowerShell extension](https://github.com/PowerShell/vscode-powershell) for VSCode (which runs in a different PS Host).
        - Impact on Completions (both in the console and in editors) - one of PowerShell's most powerful features.
        - Impact on [PSScriptAnalyzer](https://github.com/PowerShell/PSScriptAnalyzer) (which provides linting & formatting in the editor extensions).
        - Impact on [EditorSyntax](https://github.com/PowerShell/EditorSyntax) (which provides syntax highlighting with in VSCode, GitHub, and many other editors).

### CODE REVIEW

File: tools/install-powershell-readme.md [hunk 1]
```diff
@@ -66,3 +66,29 @@ bash <(wget -O - https://raw.githubusercontent.com/PowerShell/PowerShell/master/
...
+* automatically looks up latest version via git tags
+* optionally installs silently
+* optionally adds the install location to Path environment variable
+
+## Examples
+
+### Install PowerShell Core Daily Build
+
+```PowerShell
+iex "& { $(irm 'https://raw.githubusercontent.com/PowerShell/PowerShell/master/tools/install-powershell.ps1') } -daily"
```
Comment (SteveL-MSFT) [inline tools/install-powershell-readme.md]: The URL can be replaced by https://aka.ms/install-powershell.ps1 which points to the same location.

File: tools/install-powershell-readme.md [hunk 2]
```diff
@@ -66,3 +66,29 @@ bash <(wget -O - https://raw.githubusercontent.com/PowerShell/PowerShell/master/
 ### Installation To do list
 
 * Detect and wait when package manager is busy/locked? - at least Ubuntu (CentOS does this internally)
+
+# install-powershell.ps1
+
+## Features of install-powershell.ps1
+
+* can be called directly from git
```
Comment (SteveL-MSFT) [inline tools/install-powershell-readme.md]: Bullet points should start capitalized

**Review by SteveL-MSFT — CHANGES_REQUESTED** [review]

**Review by TravisEz13 — APPROVED** [review]

**Review by SteveL-MSFT — APPROVED** [review]

**Comment by SteveL-MSFT** [comment]
@kilasuit looks like you amended your commit rather than push a new commit?  This makes it hard to see what changed.  In general, you should just push new commits.  When we merge, it'll be squashed.

File: tools/install-powershell-readme.md [hunk 3]
```diff
@@ -66,3 +66,29 @@ bash <(wget -O - https://raw.githubusercontent.com/PowerShell/PowerShell/master/
...
+* Automatically looks up latest version via git tags
+* Optionally installs silently
+* Optionally adds the install location to Path environment variable
+
+## Examples
+
+### Install PowerShell Core Daily Build
+
+```PowerShell
+iex "& { $(irm 'https://aka.ms/install-powershell.ps1') } -daily"
```
Comment (iSazonov) [inline tools/install-powershell-readme.md]: Please expand aliases.

File: tools/install-powershell-readme.md [hunk 4]
```diff
@@ -66,3 +66,29 @@ bash <(wget -O - https://raw.githubusercontent.com/PowerShell/PowerShell/master/
...
+```
+### Install PowerShell Core using the MSI installer
+
+```PowerShell
+iex "& { $(irm 'https://aka.ms/install-powershell.ps1') } -UseMSI"
```
Comment (iSazonov) [inline tools/install-powershell-readme.md]: The same.

**Comment by kilasuit** [comment]
@SteveL-MSFT - noted your comment and will remember this in future as I have a habit of rebasing and rewriting commit history

**Comment by PoshChan** [comment]
@TravisEz13, this is the reminder you requested 1 hour ago

**Comment by TravisEz13** [comment]
@PoshChan Please remind me in 1 hour

**Comment by PoshChan** [comment]
@TravisEz13, this is the reminder you requested 1 hour ago

**Pull request merged**